doc/2/plugins/plugin-context/accessors/realtime/index.md

```diff
@@ -0,0 +1,117 @@
+---
+code: true
+type: page
+title: registerSubscription
+---
+
+# `realtime`
+
+Adds or removes [real-time subscriptions](/core/2/guides/essentials/real-time) from the backend.
```

Comment:
```suggestion
Manage [real-time subscriptions](/core/2/guides/essentials/real-time) from the backend.
```
For now it's only add/remove but it may evolve

Reply:
(Manages)